tasks/README.md

```diff
@@ -169,10 +171,11 @@ import os
 import json
 
 params = json.load(sys.stdin)
-sys.path.append(os.path.join(params['_installdir'], 'python_helpers/files'))
-import py_helper
+execfile(os.path.join(params['_installdir'], 'multi_task', 'files', 'py_helper.py'))
+# Alternatively use relative path
+# execfile(os.path.join(os.path.dirname(__file__), '..', '..', 'multi_task', 'files', 'py_helper.py'))
```

Comment:
`__file__` currently resolves to top level directory of  will resolve to top level of `_installdir`. This is not yet implemented? 
```
cas@cas-ThinkPad-T460p:~/working_dir/bolt/Boltdir$ bolt task run multi_task -n ah7qmsyge2sthx7.delivery.puppetlabs.net
Loading modules from /home/cas/working_dir/bolt/bolt-modules:/home/cas/working_dir/bolt/Boltdir/modules:/home/cas/working_dir/bolt/Boltdir/site:/home/cas/working_dir/bolt/modules
Starting: task multi_task on ah7qmsyge2sthx7.delivery.puppetlabs.net
Running task multi_task with '{"_task"=>"multi_task"}' on ["ah7qmsyge2sthx7.delivery.puppetlabs.net"]
Started on ah7qmsyge2sthx7.delivery.puppetlabs.net...
Command failed with exit code 1
Failed on ah7qmsyge2sthx7.delivery.puppetlabs.net:{"node":"ah7qmsyge2sthx7.delivery.puppetlabs.net","status":"failure","result":{"_output":"","_error":{"kind":"puppetlabs.tasks/task-error","issue_code":"TASK_ERROR","msg":"The task failed with exit code 1:\nTraceback (most recent call last):\n  File \"/tmp/ae75d3c4-974a-4ed4-9014-05e9cf81a942/init.py\", line 10, in <module>\n    execfile(os.path.join(os.path.dirname(__file__), '..', '..', 'multi_task', 'files', 'py_helper.py'))\nIOError: [Errno 2] No such file or directory: '/tmp/ae75d3c4-974a-4ed4-9014-05e9cf81a942/../../multi_task/files/py_helper.py'\n","details":{"exit_code":1}}}}

  The task failed with exit code 1:
  Traceback (most recent call last):
    File "/tmp/ae75d3c4-974a-4ed4-9014-05e9cf81a942/init.py", line 10, in <module>
      execfile(os.path.join(os.path.dirname(__file__), '..', '..', 'multi_task', 'files', 'py_helper.py'))
  IOError: [Errno 2] No such file or directory: '/tmp/ae75d3c4-974a-4ed4-9014-05e9cf81a942/../../multi_task/files/py_helper.py'
 
  {
  }
Finished: task multi_task with 1 failure in 0.68 sec
Failed on 1 node: ah7qmsyge2sthx7.delivery.puppetlabs.net
Ran on 1 node in 0.74 seconds

```

Reply:
Nevermind. BOLT-931